docker/README.md

```diff
@@ -1,18 +1,28 @@
 # Get docker images
 
-You can either build the images locally, or get our "latest stable" version from the Docker hub. We recommend using the Docker hub option.
+You can either build the images locally, or get the images from the Docker hub. We recommend using the Docker hub option.
 
 ## Download images from the Docker hub
 
+**openSUSE images**
```

Comment:
Yeah, I meant for table of contents, but there for markdown AFAIK there is no automatic way to do it (similar to :toc: as in asciidoc). So don't bother with it now. When it will be available, or this readme is converted to asciidoc, there will be one. I don't favour manually created ToCs.

And in the meantime, github generates direct links for markdown headers.